CCS/3. Tutorials/5. 使用 TensorFlow Inception V3 訓練影像辨識模型、生成推論引擎.md

```diff
@@ -1 +1,154 @@
+
+Model training requires a lot of computing resources to obtain a model with a good identification effect. TWCC provides you with a container solution that uses GPU resources for computing and has the ability to quickly generate models.
+
+
+This article will teach step by step how to uses [TWCC Interactive Container](https://www.twcc.ai/doc?page=container), through GPU resources<sup>[1][2]</sup> and with the collocation of default storage system –[Hyper File System (HFS)](https://www.twcc.ai/doc?page=hfs) as a storage space for training data and models, using [TensorFlow Inception V3](https://www.tensorflow.org/api_docs/python/tf/keras/applications/InceptionV3) convolutional neural network architecture, and [CIFAR-10 dataset](https://www.cs.toronto.edu/~kriz/cifar.html) carry out cat and dog image recognition model training, generate an inference engine, and provide external image recognition services.
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:**
+- [1] A container with up to 8 GPUs can be created, please refer to [<ins>Specifications and pricing</ins>](https://www.twcc.ai/doc?page=price#%E5%AE%B9%E5%99%A8%E9%81%8B%E7%AE%97%E6%9C%8D%E5%8B%99-Container-Compute-Service-CCS).
+- [2] If you need to use resources with 8 GPUs or more, you can use the service of [<ins>TWCC Taiwania 2 (HPC CLI)</ins>](https://www.twcc.ai/doc?page=hpc_cli) to finish the work.
+:::
+
+
+## Part 1. Image recognition model training
+
+### Step 1. Sign in TWCC
+
+If you don’t have an account yet, please refer to [Register for an TWCC account](https://www.twcc.ai/doc?page=register_account).
+
+### Step 2. Create a TWCC Interactive Container
+
+Please refer to [Interactive Container](https://man.twcc.ai/s/SJlZnSOaN#%E5%BB%BA%E7%AB%8B%E9%96%8B%E7%99%BC%E5%9E%8B%E5%AE%B9%E5%99%A8) and based on the following settings, create a Interactive Container in TWCC:
+
+```
+Image file type:     TensorFlow 18.12
+Image file version:  support Python 2
+Basic Configuration: model c.super 
+```
+
+### Step 3. Connect to the container, download the training program
+
+- See [Connection method](https://www.twcc.ai/doc?page=container#連線使用方式), and using Jupyter Notebook or SSH to connect to the default storage space of the container
+
+
+- Enter the following command to download the framework program of [TWCC GitHub Inception v3](https://github.com/TW-NCHC/AI-Services/tree/V3Training) image model training to the container
+
+ ```bash
+$ git clone https://github.com/TW-NCHC/AI-Services.git
+ ```
+
+
+  
+### Step 4. Conduct AI model training
+ 
+ - Enter the "**Tutorial_Three**" directory
+ 
+ ```bash
+ $ cd AI-Services/Tutorial_Three
+ ``` 
+ 
+ - Perform model training
+ 
+ ```bash
+$ bash V3_training.sh --path ./cifar-10-python.tar.gz
+  ``` 
+
+- The following message in terminal, indicating that the model is about to be trained
```

Comment:
[SUG]

1. in terminal > in the terminal
1. , indicating....trained > indicates that the training job is about to start.
2. plz add a period, thx!